src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,41 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  gap: $gap;
+  margin: auto;
+  padding: $calendar-padding;
+
+  @each $day, $number in $days-of-week {
+    &--start-day--#{$day} &__item:first-child {
+      margin-left: (($item-size + $gap) * ($number - 1));
+    }
+  }
+
+  &__item {
+    line-height: $item-size;
+    @include square($item-size);
+    text-align: center;
+    background-color: #eee;
+    border: 1px solid #000;
+    transition: all 0.5s;
```

Comment:
it's better not use transition-property: all. Specify what exactly you would like to change